spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -147,9 +242,32 @@ def get_success(self, observations):
 
 if __name__ == "__main__":
     spot = Spot("RealGazeEnv")
-    parser = get_default_parser()
-    parser.add_argument("--target-object", "-t")
-    args = parser.parse_args()
-    config = construct_config(opts=args.opts)
+    args = parse_arguments()
+    config = construct_config_for_gaze(opts=args.opts)
+    max_episode_steps = (
+        args.max_episode_steps
+        if args.max_episode_steps is not None
+        else config.MAX_EPISODE_STEPS
+    )
+    config = update_config_for_multiple_gaze(
+        config, dont_pick_up=args.dont_pick_up, max_episode_steps=max_episode_steps
```

Comment:
could you please point me where is `dont_pick_up` being used in Spot code?

Reply:
It is over here : https://github.com/facebookresearch/spot-sim2real/blob/86b69d8306d65a20c09aaccc91832b971b9e1207/spot_rl_experiments/spot_rl/envs/base_env.py#L263 
Inside base_env::step()